raw_package/index.template.html

```diff
@@ -8,6 +8,7 @@
 <esphome-main
   version="{{ version }}"
   docsLink="{{ docs_link }}"
+  {% if streamer_mode %}streamerMode="true"{% end %}
```

Comment:
Boolean properties are set to True based on their existence. 
```suggestion
  {% if streamer_mode %}streamerMode{% end %}
```